packages/edit-navigation/src/hooks/use-navigation-editor-root-block.js

```diff
@@ -0,0 +1,25 @@
+/**
+ * WordPress dependencies
+ */
+import { useSelect } from '@wordpress/data';
+import { store as blockEditorStore } from '@wordpress/block-editor';
+
+/**
+ * Internal dependencies
+ */
+
+const useNavigationEditorRootBlock = () => {
```

Comment:
This is probably small enough to be inlined in the InserterSidebar component.

Reply:
I think it's used in x2 places though...

Reply:
Oh, good point, ignore this then!